tests/kubectl_test.go

```diff
@@ -108,10 +104,10 @@ var _ = Describe("[sig-compute]oc/kubectl integration", decorators.SigCompute, f
 			vm = libvmi.NewVirtualMachine(libvmifact.NewCirros(
 				libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
 				libvmi.WithNetwork(v1.DefaultPodNetwork()),
-			))
+			), libvmi.WithRunStrategy(v1.RunStrategyAlways))
 			vm, err = virtCli.VirtualMachine(testsuite.GetTestNamespace(vm)).Create(context.Background(), vm, metav1.CreateOptions{})
 			Expect(err).NotTo(HaveOccurred())
-			tests.StartVirtualMachine(vm)
+			Eventually(ThisVM(vm)).WithTimeout(300 * time.Second).WithPolling(time.Second).Should(BeReady())
```

Comment:
Do we actually need to wait for for the VM to be ready?

Reply:
The tests below expect it to be running, otherwise they will fail per what I see.

Reply:
Running != Ready

Reply:
The documentation says differently: https://github.com/kubevirt/kubevirt/blob/ec14523ae05a7053516b0a8bd2e229cf9d96500c/staging/src/kubevirt.io/api/core/v1/types.go#L1625

Is it wrong?

Reply:
**is running and ready** , I think here we should express the intention and so only assert the VMI exists (there might be need to adjust test)

Reply:
I do not understand your feedback here.

You are saying it is wrong to check this VM status to figure out if a VM is running?

Reply:
@EdDev This is indeed correct, we can shorten the test time just be checking that the VM object is created. It doesn't have to be in ready state because the test checks existence of the columns, and not the VM state

Reply:
@Barakmor1 I think you can replace `BeReady()` with `BeCreated()`

Reply:
@enp0s3 , how will the test on line 135 will pass then? It expects a VMI per what I understand, and that means that the VMI needs to exist, which is expressed in the VM status as `ready`.

Reply:
@xpivarc @enp0s3 I think Eddy is correct. It seems like test_id:3466 verifies that the VMI is ready [here](https://github.com/kubevirt/kubevirt/blob/070129537bc2a567a19b6a1db89ae31959128b78/tests/kubectl_test.go#L159)